test/unit/test_db_helpers.py

```diff
@@ -1,11 +1,12 @@
 """Unit tests for db_helpers module."""
-from unittest.mock import Mock
-import pytest
+import pyodbc
```

Comment:
I'm not sure why `pyodbc` moved places here, but it should be in the group with the third-party imports, rather than the standard library ones.